test/init_tracer.js

```diff
@@ -156,4 +155,23 @@ describe('initTracer', () => {
         assert.equal(tracer._metrics._factory, metrics);
         assert.equal(tracer._tags['x'], 'y');
     });
+
+    describe("upsampling", () => {
+        let config = {
+            serviceName: 'test-service'
+        };
+
+        it('should default to false', () => {
+            let tracer = initTracer(config);
```

Comment:
Why not use `const`? This is present in a number of other places, but will omit, hereafter.